pkg/logging/logger.go

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logging
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+)
+
+type loggerKey struct{}
```

Comment:
Initialize the logger here:
```
var logger *zap.SugaredLogger

type loggerKey struct{}

func init() {
	zapLogger, _ := zap.NewProduction()
	defer zapLogger.Sync()
	logger = zapLogger.Sugar()
}

func WithLogger(ctx context.Context) context.Context {
	return context.WithValue(ctx, loggerKey{}, logger)
}
```
and upstream you can change to:
```
ctx := logging.WithLogger(context.Background())
```

Reply:
Addressed ✅